packages/llvm.rb

```diff
@@ -1,276 +1,104 @@
 require 'package'
 
 class Llvm < Package
-  description 'The LLVM Project is a collection of modular and reusable compiler and toolchain technologies. The optional packages clang, lld, lldb, polly, compiler-rt, libcxx, libcxxabi and openmp are included.'
+  description 'The LLVM Project is a collection of modular and reusable compiler and toolchain technologies. The optional packages clang, lld, lldb, polly, compiler-rt, libcxx, libcxxabi, and openmp are included.'
   homepage 'http://llvm.org/'
-  version '10.0.0'
+  @_ver = '11.1.0'
+  version @_ver
   compatibility 'all'
-  source_url 'https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/llvm-10.0.0.src.tar.xz'
-  source_sha256 'df83a44b3a9a71029049ec101fb0077ecbbdf5fe41e395215025779099a98fdf'
+  source_url "https://github.com/llvm/llvm-project/archive/llvmorg-#{@_ver}-rc1.tar.gz"
```

Comment:
`version = @_ver + '-rc1'`?

Reply:
I was leaving it to wait for non-RC, but you're right.